README.md

```diff
@@ -99,94 +89,98 @@ The tests directory (augur-core/tests) contain tests and test fixtures to test t
 
-### Example Updating augur-contracts
-This deployment will generate artifacts in the ./output/contracts directory. To merge these changes into augur-contracts, there are scripts located in that repository. *When running deploymen locally this step is not done automatically*. To include these changes you can work from within your checked out augur-contracts.
+Augur's smart contracts are organized into four folders:
+- `source/contracts/factories`: Constructors for universes, markets, reporting windows, etc.
+- `source/contracts/libraries`: Data structures used elsewhere in the source code.
+- `source/contracts/reporting`: Creation and manipulation of universes, markets, reporting windows, and reporting-related tokens.
+- `source/contracts/trading`: Functions to create, view, and fill orders, to issue and close out complete sets of shares, and for traders to claim proceeds after markets are closed.
 
-```
-# Merge the local changes into the contracts repo
-augur-contracts % SOURCE="path/to/augur-core/output/contracts" BRANCH=master npm run update-contracts
+## Additional notes
 
-# Update the augur-contracts version as necessary.
-# For dev builds, increment the pre-release number.
-# For patch releases, increment the patch number of the release build being patched. E.g. if 4.6.0 was released, and we are on 4.7.0-6, and we want to do a patch of 4.6, use a 'patch' release to increment to 4.6.1
-# For release builds, set the version to be the correct major/minor version, e.g. for dev build v4.7.0-10, the release version would be v4.7.0
-augur-contracts % npm version [<version>, major,minor,patch, prerelease]
-augur-contracts % git push && git push --tags
-# Publish th NPM, if this is a pre-release tag, deploy it to the dev channel. This dev channel is the default for version published from CI.
-augur-contracts % npm publish [--tag dev]
-```
+### General information about Augur
 
-## Additional notes
+- [A Roadmap For Augur and What's Next](https://medium.com/@AugurProject/a-roadmap-for-augur-and-whats-next-930fe6c7f75a)
+- [Augur Master Plan](https://medium.com/@AugurProject/augur-master-plan-42dda65a3e3d)
+
+### Terminology
+
+[Augur Terminology](http://blog.augur.net/faq/all-terms/)
+
+### Security model
+
+The security model is based on the idea that reporters will want to invest in a market where honesty prevails, and will not trust fraudulent markets. For example, assume that there exist two almost identical universes: an honest universe, where the factually correct answer is the predominant answer, and a fraudulent universe, where incorrect answers are predominant in order to cheat the system. The REP in the fraudulent universe will approach 0, and the REP in the honest universe will approach the theoretical yield. This phenomenon occurs because reporters do not want to trade on markets in a dishonest universe. Therefore, they will not create markets in a dishonest universe, and the REP holders will not receive any market fees.
```

Comment:
"reporters do not want to trade" should be "traders do not want to trade"

Reply:
Larger point: should we just remove this abbreviated section on the security model?  I'm not a huge fan of having this stuff both here _and_ in the whitepaper.

Reply:
Yeah, I'm all for deleting this section.  I didn't realize this was a move, not an addition.  👍 to delete.